hypothesis-python/tests/typing_extensions/test_backported_types.py

```diff
@@ -78,3 +78,10 @@ def test_defaultdict(ex):
     assume(ex)
     assert all(isinstance(elem, int) for elem in ex)
     assert all(isinstance(elem, int) for elem in ex.values())
+
+
+def test_union_of_Literals():
+    assert from_type(Union[Literal["hamster"], Literal["bunny"]]).example() in (
```

Comment:
Let's also test that `Union[Literal["hamster"], Literal["bunny"]]` produce the same result as `Literal["hamster", "bunny"]`

Reply:
In that case, let's expand `test_typing_extensions_Literal_nested(data)` instead of writing a whole new test.

Reply:
@Zac-HD I'm not sure exactly what you have in mind for how to expand that test. Maybe I should leave it to you.

Reply:
Yep, I'm happy to take this one :grin: